awx/main/migrations/_rbac.py

```diff
@@ -35,7 +38,186 @@ def create_roles(apps, schema_editor):
                 obj.save()
 
 
+def delete_all_user_roles(apps, schema_editor):
```

Comment:
After we merge this PR, we should go clean up all these old unnecessary functions from previous RBAC migration; this one harkens back to Ansible Tower's 3.3.0 timeline, so it's quite old now.